app/presenters/help_page_presenter.rb

```diff
@@ -0,0 +1,2 @@
+class HelpPagePresenter < GovspeakPresenter
```

Comment:
The naming, i agree with, but overall, if we want to clean this up, perhaps we should be putting the tickets in now?